src/main/java/meowmeow/TaskList.java

```diff
@@ -0,0 +1,167 @@
+package meowmeow;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import meowmeow.events.Deadline;
+import meowmeow.events.Event;
+import meowmeow.events.Task;
+import meowmeow.events.ToDo;
+
+public class TaskList {
+    //Initialising task array for list cmd
+    private static int numOfInputs = 0;
+    private final ArrayList<Task> taskList;
+    private final Storage storage;
+
+    TaskList(ArrayList<Task> loadSave, Storage storage) {
+        this.taskList = loadSave;
+        this.storage = storage;
+        numOfInputs = loadSave.size();
+    }
+
+    public ArrayList<Task> getTaskList() {
+        return taskList;
+    }
+
+    public void printTaskList() {
+        System.out.println("Here are your tasks =0w0=");
+        Task task;
+
+        for (int i = 0; i < numOfInputs; i++) {
+            task = taskList.get(i);
+
+            if (task.isDone()) { //task is done
+                System.out.println((i + 1) + ". " + task);
+
+            } else { //task is not done
+                System.out.println((i + 1) + ". " + task);
+            }
```

Comment:
Is there a need for this if-else statement?
```suggestion
            System.out.println((i + 1) + ". " + task);
```